modules/tensor_mechanics/doc/content/source/materials/ADRankTwoCartesianComponent.md

```diff
@@ -0,0 +1,34 @@
+# Rank Two Tensor Component
+
+!syntax description /Materials/ADRankTwoCartesianComponent
+
+## Description
+
+This is an ADMaterial model used to extract components of a rank-2 tensor in a
+Cartesian coordinate system. This can be used regardless of the coordinate
+system used by the model.
+
+This ADMaterial model is used by
+[TensorMechanics/Master](/Modules/TensorMechanics/Master/index.md)
+automatically, but can also be called directly by the user. This class provides
```

Comment:
automatically when stress components are requested in the `generate_output` parameter, but...

Reply:
called directly -> set up directly